fluid/PaddleRec/word2vec/train.py

```diff
@@ -261,123 +193,42 @@ def train(args):
     if not os.path.isdir(args.model_output_dir):
         os.mkdir(args.model_output_dir)
 
-    filelist = GetFileList(args.train_data_path)
-    word2vec_reader = None
-    if args.is_local or os.getenv("PADDLE_IS_LOCAL", "1") == "1":
-        word2vec_reader = reader.Word2VecReader(
-            args.dict_path, args.train_data_path, filelist, 0, 1)
-    else:
-        trainer_id = int(os.environ["PADDLE_TRAINER_ID"])
-        trainer_num = int(os.environ["PADDLE_TRAINERS"])
-        word2vec_reader = reader.Word2VecReader(args.dict_path,
-                                                args.train_data_path, filelist,
-                                                trainer_id, trainer_num)
+    filelist = GetFileList(args.train_data_dir)
+    word2vec_reader = reader.Word2VecReader(args.dict_path, args.train_data_dir,
+                                            filelist, 0, 1)
 
     logger.info("dict_size: {}".format(word2vec_reader.dict_size))
+    #update id_frequency
+    sum_val = 0.0
+    id_frequencys_pow = []
+    for id_x in range(len(word2vec_reader.id_frequencys)):
+        id_frequencys_pow.append(
+            math.pow(word2vec_reader.id_frequencys[id_x], 0.75))
+        sum_val += id_frequencys_pow[id_x]
+    id_frequencys_pow = [val / sum_val for val in id_frequencys_pow]
```

Comment:
Maybe it's more clear to use numpy?
```suggestion
    id_frequencys_pow = np.power(word2vec_reader.id_frequencys, 0.75)
    id_frequencys_pow /= np.sum(id_frequencys_pow)
```